architecture/topics/auth_enabled_registry.adoc

```diff
@@ -0,0 +1,114 @@
+// This assembly is included in the following assemblies:
+//
+// <architecture/infrastructure_components/image_registry#auth-enabled-registry.adoc>
+//<install_config/registry/index#auth-enabled-registry.adoc>
+
+// This assembly can be included from other assemblies using the following
+// include statement:
+
+// The following line is necessary to allow assemblies be included in other
+// assemblies. It restores the `context` variable to its previous state.
+:parent-context-of-creating-build-inputs: {context}
+
+// The file name and the ID are based on the assembly title.
+// For example:
+// * file name: assembly_my-assembly-a.adoc
+// * ID: [id='assembly_my-assembly-a_{context}']
+// * Title: = My assembly A
+
+//Concept
+All container images available through the Red Hat Container Catalog are hosted
+on an image registry, `registry.access.redhat.com`. The Red Hat Container Catalog
+is moving from `registry.access.redhat.com` to `registry.redhat.io`.
+
+The new registry, `registry.redhat.io`, requires authentication for access to
+images and hosted content on {product-title}. Following the move to the new
```

Comment:
"Following the move to the new registry, the existing registry will be available for a period of time."

Can we be more specific? If not, I'd remove it.

Reply:
This is a tough one. I left it intentionally vague. We need to let readers know that the existing registry will eventually be removed. I could maybe rewrite it. 

The goal is 12 months. But if we make that statement and it changes, we will be responsible for maintaining it and could run into further problems.